src/BlazorDB/wwwroot/blazorDB.js

```diff
@@ -132,6 +132,21 @@ window.blazorDB = {
             });
         });
     },
+    where: function(dotnetReference, transaction, dbName, storeName, condition) {
+        return new Promise((resolve, reject) => {
+            window.blazorDB.getDb(dbName).then(db => {
+                db[storeName].where(JSON.parse(condition)).toArray(items => {
```

Comment:
Can you use this approach instead?  Just to be consistent

https://github.com/nwestfall/BlazorDB/blob/1ef4e14d8eea7ce0f47e7430eae63fa23f6aee44/src/BlazorDB/wwwroot/blazorDB.js#L123

Reply:
I will take a look at it.